pkg/app/piped/controller/scheduler.go

```diff
@@ -537,6 +538,23 @@ func (s *scheduler) executeStage(sig executor.StopSignal, ps model.PipelineStage
 		return model.StageStatus_STAGE_FAILURE
 	}
 
+	// Skip the stage if needed based on the skip config.
+	skip, err := s.checkSkipStage(sig.Context(), input)
+	if err != nil {
+		lp.Errorf("failed to check whether skipping the stage", zap.Error(err))
```

Comment:
```suggestion
		lp.Errorf("failed to check whether skipping the stage: %v", err.Error())
```

Reply:
@t-kikuc This `lp` is log persister, a printer that prints log as stage log and is viewable from web UI. While the `logger` is a zap logger, which prints the execution log in stdout of the piped running container. So basically, we can't use the `zap` format with log persister and vice versa. Please refer other place to understand how to use those loggers